src/rules/indentation.js

```diff
@@ -16,60 +16,86 @@ var availableConfigs = {
   'but': 2
 };
 
-var errors = [];
+var availableConfigs = _.merge({}, defaultConfig, {
+  // The values here are unused by the config parsing logic.
+  'feature tag': -1,
+  'scenario tag': -1
+});
 
-function test(parsedLocation, configuration, type) {
-    // location.column is 1 index based so, when we compare with the expected indentation we need to subtract 1
-  if (--parsedLocation.column !== configuration[type]) {
-    errors.push({message: 'Wrong indentation for "' + type +
-                          '", expected indentation level of ' + configuration[type] +
-                          ', but got ' + parsedLocation.column,
-                 rule   : rule,
-                 line   : parsedLocation.line});
+function mergeConfiguration(configuration) {
+  var mergedConfiguration = _.merge({}, defaultConfig, configuration);
+  if (!Object.prototype.hasOwnProperty.call(mergedConfiguration, 'feature tag')) {
```

Comment:
btw I think this could be simplified to `if (!mergedConfiguration.hasOwnProperty('feature tag')) {`

Reply:
Yeah, absolutely, my one and only js project enforced this rule https://eslint.org/docs/rules/no-prototype-builtins, and I've obviously internalised it!

Reply:
Oh I see, I don't mind leaving it like this since there is a legitimate case where the more verbose code is more robust